### ISSUE TITLE
The default font weight in vscode is a bit bulk

### ISSUE THREAD

**Issue by TZECHIN6** [githubnext/monaspace#182]
![image](https://github.com/githubnext/monaspace/assets/66129247/c0244724-fbcb-4b3b-99ed-4f94abb63c87)

Why does the font-weight seem that heavy (bold) in vscode? Below is my setting, I haven't set any fontweight modifier... should be the default regular one.

```
 "editor.fontFamily": "'Monaspace Neon', monospace",
 "editor.fontLigatures": "'calt', 'liga', 'dlig', 'ss01', 'ss02', 'ss03'",
```

for comparison, below are the same config with `Source code pro`
![image](https://github.com/githubnext/monaspace/assets/66129247/d02251fe-ce32-4bfc-a991-9b2cea4bc493)

**Comment by hossinhabeeb119** [comment]
> ![image](https://github.com/githubnext/monaspace/assets/66129247/c0244724-fbcb-4b3b-99ed-4f94abb63c87)
> 
> 
> 
> 
> 
> Why does the font-weight seem that heavy (bold) in vscode? Below is my setting, I haven't set any fontweight modifier... should be the default regular one.
> 
> 
> 
> ```
> 
>  "editor.fontFamily": "'Monaspace Neon', monospace",
> 
>  "editor.fontLigatures": "'calt', 'liga', 'dlig', 'ss01', 'ss02', 'ss03'",
> 
> ```
> 
> 
> 
> for comparison, below are the same config with `Source code pro`
> to
> ![image](https://github.com/githubnext/monaspace/assets/66129247/d02251fe-ce32-4bfc-a991-9b2cea4bc493)
> 
>

**Comment by nmsmith** [comment]
I agree. This font is bolder than all of the other monospace fonts I've used before. IMO, it's not really usable at this level of boldness.

Note: This is not a VSCode issue. It appears just as bold in other apps.

**Comment by heathercran** [comment]
This is always a matter of preference, which is why we supply fonts in several weight options. Try setting fontFamily to `'MonaspaceNeon-Light'` and see if that looks more natural to you!
